src/test/mir-opt/const_prop/discriminant.rs

```diff
@@ -31,7 +31,7 @@ fn main() {
 // START rustc.main.ConstProp.after.mir
 //  bb0: {
 //      ...
-//      _3 = const Scalar(0x01) : std::option::Option<bool>;
+//      _3 = const {0x01 : std::option::Option::<bool>};
```

Comment:
The type's syntax is wrong now (it should be `Option<bool>` not `Option::<bool>`), but also I'm guessing you're not printing ADTs yet?

Reply:
yea, we don't print any aggregates but single element arrays

Reply:
I fixed this